tools/openvino_dev/requirements.txt

```diff
@@ -1,28 +1,7 @@
-defusedxml>=0.7.1
-scipy~=1.5.4
 jstyleson~=0.0.2
-numpy>=1.16.6,<1.20
 addict>=2.4.0
-pandas~=1.1.5
 hyperopt~=0.1.2
-networkx~=2.5
-tqdm>=4.54.1
 texttable~=1.6.3
-py-cpuinfo>=7.0.0
-PyYAML>=5.4.1
-pillow>=8.1.2
-scikit-image>=0.17.2
-scikit-learn>=0.24.1
 yamlloader>=0.5
-shapely>=1.7.1
-nibabel>=3.2.1
-pydicom>=2.1.2
-sentencepiece>=0.1.95
-tokenizers>=0.10.1
-editdistance>=0.5.3
-parasail>=1.2.4
-fast-ctc-decode>=0.2.5
-rawpy>=0.16.0
-nltk>=3.5
 opencv-python==4.5.*
 progress>=1.5
```

Comment:
What is the origin of other requirements?

Reply:
benchnmark_app and pot

Reply:
cleaned up this list to get all requirements from tools directly

Reply:
this file is empty now, can we drop it?

Reply:
we have plans to add other dependencies, like telemetry

Reply:
why is it a common dependency? It should be dependency of particular tool anyway